app/controllers/submissions_controller.rb

```diff
@@ -17,7 +17,7 @@ def destroy
       render json: {message: "Cannot find submission"}, status: :not_found
     else 
       if @submission.destroy
-        render json: {deleted: true}, status: :success
+        render json: {deleted: true}
```

Comment:
checkout [2.2.13.4 The :status Option](http://guides.rubyonrails.org/layouts_and_rendering.html#using-render) to know which status codes are available. And remember, always...

```
CONVENTION OVER CONFIGURATION
```